src/app/data-planes/components/DataPlaneList.vue

```diff
@@ -407,7 +405,7 @@ async function parseData(dataPlaneOverview: DataPlaneOverview) {
   let zoneRoute: RouteLocationNamedRaw | undefined
   if (zone !== undefined) {
     zoneRoute = {
-      name: 'zones',
+      name: 'zone-list-view',
       query: {
         ns: zone,
```

Comment:
Do you think we should get rid of the `ns` param name entirely and use `zone` or something (I think thats what we did for policies). `ns` to me means `namespace` which doesn't make sense to me here. lmk

Reply:
This will be handled in the follow-up PR I have prepared for this. It removes all routes using the `ns` param and targets detail view routes instead.

Reply:
Gotcha